src/app/container/descriptors/descriptors.component.ts

```diff
@@ -25,8 +23,8 @@ import { EntryFileSelector } from '../../shared/selectors/entry-file-selector';
 import { GA4GHService, ToolDescriptor, ToolFile } from '../../shared/swagger';
 import { Tag } from '../../shared/swagger/model/tag';
 import { ToolQuery } from '../../shared/tool/tool.query';
-import { FilesService } from '../../workflow/files/state/files.service';
 import { FilesQuery } from '../../workflow/files/state/files.query';
+import { FilesService } from '../../workflow/files/state/files.service';
```

Comment:
Why is this moving around, does prettier handle import ordering?

Reply:
It's a linter thing, not prettier. I have it turned on for mine because it's the "correct" way to do imports. If my changes touch imports, mine will correct it. Our previous co-ops didn't use an IDE that supports it so it seemed unreasonable for me to request changes every time their imports weren't in the correct order.